taxi/forms.py

```diff
@@ -10,6 +10,7 @@ class CarForm(forms.ModelForm):
     drivers = forms.ModelMultipleChoiceField(
         queryset=get_user_model().objects.all(),
```

Comment:
Using `get_user_model().objects.all()` might not be appropriate if `Driver` is a specific subclass of the user model. Consider filtering the queryset to include only instances of `Driver` if necessary.